README.md

```diff
@@ -32,13 +32,51 @@ conda env remove -n sleap-anipose
 ```
 
 ## Usage
+
+**From the command line**
+
+1. Activate the conda environment (assuming you installed via conda).
+```
+conda activate sleap-anipose
+```
+
+2. Generate a calibration board matching the details of the board presented before the cameras. 
+
+```
+slap-write_board --fname board.toml --board_width 8 --board_height 11 --square_length 24.0 marker_length --18.5 --marker_bits 4 dict_size 1000
+```
+
+3. If you do not have a calibration board already, you can have the calibration board drawn and saved to a pdf file from the file made in step 2. 
+
+```
+slap-draw_board --fname board.pdf --board board.toml 
+```
+
+4. Calibrate your camera setup by displaying the calibration board to all cameras. Make sure to do this before running any experiments. See [`CALIBRATION_GUIDE.md`](docs/CALIBRATION_GUIDE.md) for more information.
+
+5. Track your data using SLEAP. Make sure that animal tracks are matching across multiple views. See [`SLEAP_GUIDE.md`](docs/SLEAP_GUIDE.md) for more information.
+
+6. Set up your data according to [`FOLDER_STRUCTURE.md`](docs/FOLDER_STRUCTURE.md).
+
+7. Generate the calibration for your cameras:
+```
+slap-calibrate --session date_directory/session_directory --board board.toml --save_calib True --save_folder date_directory/session_directory --save_metadata True --histogram True -reproj_imgs True --save_hist True --save_reproj_imgs True
```

Comment:
if you have boolean flags, just use the `is_flag=True` option in the decorator so you can make the CLI less verbose ([example](https://click.palletsprojects.com/en/8.1.x/options/#:~:text=For%20single%20option%20boolean%20flags%2C%20the%20default%20remains%20hidden%20if%20the%20default%20value%20is%20False.))

Reply:
Corrected